package.json

```
@@ -58,6 +59,7 @@
   "devDependencies": {
     "elm": "0.19.1-3",
     "elm-format": "0.8.3",
+    "elm-review": "^2.3.3",
```

Can we pin this dep:

```suggestion
    "elm-review": "2.3.3",
```